systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -683,9 +683,10 @@ public void testWatchingOtherNamespace() throws InterruptedException {
         assertThat(topics, not(hasItems("my-topic")));
         String origNamespace = kubeClient.namespace("topic-operator-namespace");
         kubeClient.create(new File("../examples/topic/kafka-topic.yaml"));
-        Thread.sleep(10_000);
-        kubeClient.namespace(origNamespace);
-        topics = listTopicsUsingPodCLI(CLUSTER_NAME, 0);
-        assertThat(topics, hasItems("my-topic"));
+        TestUtils.waitFor("wait for 'my-topic' to be created in Kafka", 120000, 5000, () -> {
+            kubeClient.namespace(origNamespace);
+            List<String> topics2 = listTopicsUsingPodCLI(CLUSTER_NAME, 0);
```

Comment:
We can use topics instead of topics2 here as well because it is in a different scope right?